cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -126,13 +128,13 @@ import           Cardano.Api.TxInMode
 data LocalNodeClientProtocols block point tip tx txerr query m =
      LocalNodeClientProtocols {
        localChainSyncClient
-         :: LocalChainSyncClient block point tip m
+         :: NodeToClientVersion -> LocalChainSyncClient block point tip m
 
      , localTxSubmissionClient
-         :: Maybe (LocalTxSubmissionClient tx txerr          m ())
+         :: Maybe (NodeToClientVersion -> LocalTxSubmissionClient tx txerr          m ())
 
      , localStateQueryClient
-         :: Maybe (LocalStateQueryClient   block point query m ())
+         :: Maybe (NodeToClientVersion -> LocalStateQueryClient   block point query m ())
```

Comment:
I don't understand why we are passing it in here. If we're passing it in to the entire `LocalNodeClientProtocolsForBlock` / `LocalNodeClientProtocolsInMode` then we do not need to pass it into individual handlers.